ros2/dashing/ros-base/Dockerfile

```
@@ -10,5 +10,15 @@ RUN apt update -qq && apt install -y -q \
     gnupg2 \
     && rm -rf /var/lib/apt/lists/*
 
+# Add AS Ubuntu repo
+RUN echo "deb [trusted=yes] https://s3.amazonaws.com/autonomoustuff-repo/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/autonomoustuff-public.list
+
+# Install AS Common
+RUN apt-get update -qq && apt-get install -y -q \
+    libas-common
```

Please add:
```suggestion
    libas-common \
    && rm -rf /var/lib/apt/lists/*
```
This clears the `apt` cache in this Docker layer so there isn't extra bloat in the container image.